scripts/mergeAOSP.sh

```diff
@@ -59,4 +59,10 @@ resetSubFolder "./compose/**/desktopMain/**"
 resetSubFolder "./compose/**/skikoTest/**"	
 resetSubFolder "./compose/**/desktopTest/**"	
 resetSubFolder "./compose/desktop/**"	
+# specify folders that doesn't exist in JetBrains fork, because Git might treat some files as renames to these folders
+resetSubFolder "./compose/**/linuxx64StubsMain/**"
+resetSubFolder "./compose/**/linuxx64StubsTest/**"
+resetSubFolder "./compose/**/jvmStubsMain/**"
+resetSubFolder "./compose/**/jvmStubsTest/**"
+resetSubFolder "./compose/**/nonAndroidMain/**"
```

Comment:
It's not part of stabs, please move it into appropriate place/section

Reply:
The comment says about subfolders that does not exist in fork not about stubs